src/client/components/voyage/InputField.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class InputField extends Component {
```

Comment:
This component clearly assumes that you are going to input numbers (type number and default value 0). How about calling it `InputFieldNumber`?